packages/server/postgres/migrations/1624862201216_addAtlassianAuthTable.ts

```diff
@@ -0,0 +1,84 @@
+import {ColumnDefinitions, MigrationBuilder} from 'node-pg-migrate'
+import {Client} from 'pg'
+import {r} from 'rethinkdb-ts'
+import {parse} from 'url'
+import getPgConfig from '../getPgConfig'
+import {insertAtlassianAuthsQuery} from '../queries/generated/insertAtlassianAuthsQuery'
+import AtlassianManager from 'parabol-client/utils/AtlassianManager'
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+const connectRethinkDB = async () => {
+  const {hostname: host, port, path} = parse(process.env.RETHINKDB_URL)
+  await r.connectPool({
+    host,
+    port: parseInt(port, 10),
+    db: path.split('/')[1]
+  })
+}
+
+export async function up(): Promise<void> {
+  await connectRethinkDB()
+  const client = new Client(getPgConfig())
+  await client.connect()
+
+  await client.query(`
+    CREATE TABLE IF NOT EXISTS "AtlassianAuth" (
+      "accessToken" VARCHAR(2600) NOT NULL,
+      "refreshToken" VARCHAR(2600) NOT NULL,
+      "createdAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "updatedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "isActive" BOOLEAN DEFAULT TRUE NOT NULL,
+      "jiraSearchQueries" JSONB[] NOT NULL DEFAULT '{}',
+      "cloudIds" VARCHAR(120)[] NOT NULL DEFAULT '{}',
+      "scope" VARCHAR(240) NOT NULL,
+      "accountId" VARCHAR(120) NOT NULL,
+      "teamId" VARCHAR(120) NOT NULL,
+      "userId" VARCHAR(120) NOT NULL,
+      PRIMARY KEY ("userId", "teamId")
+    );
+  `)
+
+  const atlassianIntegrations = await r
+    .table('AtlassianAuth')
+    .filter((row) =>
+      row('accessToken')
+        .default(null)
+        .ne(null)
+    )
+    .run()
+  const auths = atlassianIntegrations.map(
+    ({
+      accessToken,
+      refreshToken,
+      createdAt,
+      updatedAt,
+      isActive,
+      jiraSearchQueries,
+      cloudIds,
+      accountId,
+      teamId,
+      userId
+    }) => ({
+      accessToken,
+      refreshToken,
+      createdAt,
+      updatedAt,
+      isActive,
+      accountId,
+      teamId,
+      userId,
+      jiraSearchQueries,
+      cloudIds,
+      scope: AtlassianManager.SCOPE
+    })
+  )
+  if (auths.length) {
+    await insertAtlassianAuthsQuery.run({auths}, client)
+  }
```

Comment:
Should this be a part of the migration or a separate post-deploy script?

Reply:
here is perfect! i'm all for trying to keep as many migrations in the pre-deploy as possible